packages/components/organisms/f-header/src/components/CountrySelectorPanel.vue

```diff
@@ -34,23 +37,26 @@
                     :key="country.key"
                     :class="$style['c-countrySelector-country']"
                     :data-test-id="[`countrySelector-countryList-${country.dataTestKey}`]">
-                    <a
-                        :data-trak='`{
-                            "trakEvent": "click",
-                            "category": "engagement",
-                            "action": "header",
-                            "label": "${country.gtm}"
-                        }`'
+                    <nav-link
+                        :data-trak="{
+                            trakEvent: 'click',
+                            category: 'engagement',
+                            action: 'header',
+                            label: `${country.gtm}`
+                        }"
                         :tabindex="isOpen ? 0 : -1"
-                        :href="country.siteUrl"
-                        :class="$style['c-countrySelector-link']">
-                        <flag-icon
-                            :country-code="country.flagKey"
-                            :class="$style['c-nav-list-icon--flag']" />
-                        <span>
-                            {{ country.localisedName }}
-                        </span>
-                    </a>
+                        :has-border-bottom="false"
+                        :is-below-mid="isBelowMid"
+                        :is-country-selector-panel="true"
```

Comment:
Rather than use a prop for this (which adds a classname), could you not add a class to this component `c-countrySelector-country-link`, that you then add styles for here in this component? (I think that should work with CSS modules, but may need to try it)

Reply:
Yes, that's fixed it thank you - I've created `c-countrySelector-country-link a` so it's declared here but only styles the link inside the `NavLink` component 😄